desktop_server/src/com/mygdx/game/server/ecs/entityfactory/UnitFactory.java

```diff
@@ -32,6 +32,7 @@ public void createEntity(
       @NonNull Client client
   ) {
     int entityId = componentFactory.createEntityId();
+    componentFactory.createUnitComponent(entityId);
```

Comment:
W sumie czy nie może być używane moveable?

Reply:
bo już mi się nie chciało zastanawiać nad tym jaki zestaw componentów definiuje nam coś co można zidentyfikować jako unita

i też można się "kłócić", że unit to osoba cecha, która polega na tym, że jak stanie na polu to je przejmuje, więc nie musi być movable